### PR TITLE
fix-typo: consecutive-word duplications

### PR DESCRIPTION
**Pull request by ryenus**
This fixes unintentional duplication(s) of words, such as the the, and
can can etc. I also take the chance to make a few wording tweaks to
avoid suspicious duplication(s).

### CODE REVIEW

**Comment by ryenus** [comment]
@derrickstolee, could you please help /allow me?

File: Documentation/howto/revert-a-faulty-merge.txt [hunk 1]
```diff
@@ -14,7 +14,7 @@ How to revert a faulty merge
 
 Alan <alan@clueserver.org> said:
 
-    I have a master branch.  We have a branch off of that that some
+    I have a master branch.  We have a branch off of it that some
```
Comment (dscho) [inline Documentation/howto/revert-a-faulty-merge.txt]: That's not a word duplication.

File: Documentation/howto/revert-branch-rebase.txt [hunk 2]
```diff
@@ -27,8 +27,8 @@ $ git checkout -b revert-c99 master
 Now I am on the 'revert-c99' branch.  Let's figure out which commit to
 revert.  I happen to know that the top of the 'master' branch is a
 merge, and its second parent (i.e. foreign commit I merged from) has
-the change I would want to undo.  Further I happen to know that that
-merge introduced 5 commits or so:
+the change I would want to undo.  Further I happen to know that there're
+5 commits or so introduced by that merge:
```
Comment (dscho) [inline Documentation/howto/revert-branch-rebase.txt]: That's not a word duplication.

File: Documentation/howto/revert-a-faulty-merge.txt (hunk 1 above)
Comment (dscho) [reply Documentation/howto/revert-a-faulty-merge.txt]: (Both of these non-word-duplications should be factored out into their own commit, and the commit message should probably make a case for the change. For example, I am not all that certain that we should edit what Alan said. [He said what he said](https://www.youtube.com/watch?v=Cc6dQPFKh28)...)

File: Documentation/howto/revert-branch-rebase.txt (hunk 2 above)
Comment (dscho) [reply Documentation/howto/revert-branch-rebase.txt]: As you can see from the header in lines 1--9 of this file, this is a verbatim copy of an email by Junio Hamano. I am not all that sure that it should be considered appropriate to edit what they said, other than fixing typos and obvious style issues. This change does not strike me as an obvious typo or style improvement...

File: Documentation/howto/revert-a-faulty-merge.txt (hunk 1 above)
Comment (ryenus) [reply Documentation/howto/revert-a-faulty-merge.txt]: Indeed

File: Documentation/howto/revert-branch-rebase.txt (hunk 2 above)
Comment (ryenus) [reply Documentation/howto/revert-branch-rebase.txt]: agree

**Comment by ryenus** [comment]
Thank you @dscho for the comments, that all make sense.
And I've updated the commit accordingly.

File: Documentation/RelNotes/2.13.0.txt [hunk 3]
```diff
@@ -38,7 +38,7 @@ UI, Workflows & Features
 
  * Deletion of a branch "foo/bar" could remove .git/refs/heads/foo
    once there no longer is any other branch whose name begins with
-   "foo/", but we didn't do so so far.  Now we do.
+   "foo/", but we didn't do so far.  Now we do.
```
Comment (Denton-L) [inline Documentation/RelNotes/2.13.0.txt]: I would argue that this was correctly written in the original. Perhaps we could change this to "but we didn't do _that_ so far" but I'm also okay with the original wording. Dunno.

Comment (ryenus) [reply Documentation/RelNotes/2.13.0.txt]: Good catch! Thank you @Denton-L :-) 

You're surely right about the correctness of the original wording, 
and I think what you suggested is good, too, and that helps make it
no longer suspicious.

**Comment by dscho** [comment]
Did you mean to `/submit` a new iteration of the patch?

**Pull request closed without merging**